test/utils/browser.js

```diff
@@ -30,32 +32,86 @@ exports.openMusic = function (options) {
   // Execute many async steps
   before(function startBrowser () {
     this.browser = wd.remote();
+    process.argv.forEach(function eachArgument (arg) {
+      // Uniquely identify a browserstack build based on timeout
+      if (arg === '40000') {
+        this.browser = wd.remote('hub.browserstack.com', 80);
+      }
+    }.bind(this));
     global.browser = this.browser;
   });
   before(function openBrowser (done) {
-    this.browser.init({browserName: 'chrome'}, done);
+    var that = this;
+    this.browser.init({
+      browserName: 'firefox',
+      name: 'Google Music Core Test Suite',
+      project: 'Google Music Core Test Suite',
+      platform: 'WIN8',
```

Comment:
The alternatives were generic windows and XP.  XP appeared to have some loading / stability issues every now and again, when I enforced windows 8 only these issues went away.